acquire/acquire.py

```diff
@@ -1217,9 +1217,13 @@ class RemoteAccess(Module):
         ("glob", "sysvol/Program Files/TeamViewer/*.log"),
         ("glob", "sysvol/Program Files (x86)/TeamViewer/*.log"),
         ("glob", "AppData/Roaming/TeamViewer/*.log", from_user_home),
-        # anydesk
+        # anydesk - Windows
         ("dir", "sysvol/ProgramData/AnyDesk"),
         ("glob", "AppData/Roaming/AnyDesk/*.trace", from_user_home),
+        ("glob", "AppData/Roaming/AnyDesk/*/*.trace", from_user_home),
+        # anydesk - Mac + Linux
+        ("glob", ".anydesk*/*.trace", from_user_home),
```

Comment:
Just checking - is it actually `.anydesk*` or should it be `.anydesk/*`?

Reply:
It should be like it is ;)

https://support.anydesk.com/knowledge/trace-files#trace-file-locations --> For the Custom Client, it could be located in `~/.anydesk_ad_<prefix>/anydesk_ad_<prefix>.trace` or for the Standard Client just `~/.anydesk/anydesk.trace`